Add blog post about rollout steps (#335)
readthedocs/website: content/posts/rollout-of-our-new-dashboard.rst

:title: Rollout of our new dashboard
:date: 2024-11-11
:description: Our new dashboard will begin rolling out to users.
              Learn what to expect and how it will affect you and your project.
:category: Changelog
:tags: dashboard
:authors: Anthony Johnson
:status: published
:image: /images/new-dashboard.png
:image_credit: Reviewing build output for a project in our new dashboard

For the last year, you may have seen notifications and hints asking you to try
out our new dashboard. This new dashboard has been a long term project to

> completely replace our still original user interface with one that is more

Reviewer: ```suggestion
completely replace our original user interface with one that is more
```

consistent and allows us to more easily build and maintain complex features.

During the next few months, we will begin the transition to our new dashboard
and will move away from our current, *but soon to be legacy*, dashboard.

If you are not yet using our new dashboard, here is what you need to know:

The new dashboard will start to become the default dashboard
    After this change, users logging in from our website will be logging into
    our new dashboard. *Users will still be able to switch back to the legacy
    dashboard if they need to*.

    - On `Read the Docs Community`_ this change will go into effect on **Nov 12th**

>     - On `Read the Docs Business`_ this change will go into effect on **Dec 10th**

Reviewer: We should mention the URL of the new dashboard here, probably.

Reviewer: ```suggestion
    - On `Read the Docs Community`_ this change will go into effect on **Nov 12th**
    - On `Read the Docs Business`_ this change will go into effect on **Dec 10th**
    
 
    To log in to the new dashboard:

    - Users of `Read the Docs Community`_ can log in at https://app.readthedocs.org/account/login
    - Users of `Read the Docs Business`_ can log in at https://app.readthedocs.com/account/login
```

> The legacy dashboard will be available until Mar 1st, 2025

Reviewer: Do we want to define a date here before we've migrated over? Feels a little ambitious, but probably good to have a deadline, and just do the work to make sure we hit it.

Author: Yeah I felt a date was fine here. The only thing we're waiting on in this period is for users to switch their daily usage to the new dashboard -- we don't need users to make content changes and we won't have any migration tasks left after Dec 10th.

We can also always extend this timeframe if users are stuck using the old dashboard for some reason.

Author: I'll call the date Mar 11th, so 3 months even from Dec 10th.

```suggestion
The legacy dashboard will be available until Mar 11th, 2025
```

    our new dashboard. During this period, our legacy dashboard will no longer
    be receiving updates or new features, but it will remain active. Our legacy
    dashboard will only be available for this limited period, after which it
    will be shut down for all users.

    To log in to the legacy dashboard:

    - Users of `Read the Docs Community`_ can log in at https://readthedocs.org/account/login
    - Users of `Read the Docs Business`_ can log in at https://readthedocs.com/account/login

How is the new dashboard different?
-----------------------------------

You should find that the new dashboard is more consistent and that navigation
around the dashboard is easier overall. New features were added for sorting and
filtering, build output is easier to browser and share, and project and version
creation is more intuitive.

The new dashboard contains many other updates and new features as well. In the
coming weeks we will be highlighting in more detail what new features were
added, interfaces that have been updated, and some of the technical changes
introduced.

Documentation page changes
~~~~~~~~~~~~~~~~~~~~~~~~~~

> Project documentation will also receive an update with these dashboard changes.
> There are some points in hosted documentation where our own dashboard is
> surfaced, such as documentation error pages, authentication forms, and
> documentation sharing forms.

Reviewer: ```suggestion
**The rendered documentation for your project will not change.**
This change primarily applies to the dashboard used for managing your documentation projects,
and not the rendered pages.

There are some points in hosted documentation where our own dashboard is
surfaced, such as documentation error pages, authentication forms, and
documentation sharing forms.
These pages will have the new look and feel.
```

I think we want to be really clear this doesn't impact their actual rendered docs, just system pages.

Author: Yeah this is worth spelling out more. I think I'd rather this is more explicit overall now though. It all feels a little vague without this, especially given we're warning the user.

:title: Rollout of our new dashboard

> :date: 2024-11-11

Reviewer 2: ```suggestion
:date: 2024-11-12
```
